pkg/odo/cli/cli.go

```diff
@@ -173,8 +174,11 @@ func NewCmdOdo(name, fullName string) *cobra.Command {
 		config.NewCmdConfiguration(config.RecommendedCommandName, util.GetFullName(fullName, config.RecommendedCommandName)),
 		preference.NewCmdPreference(preference.RecommendedCommandName, util.GetFullName(fullName, preference.RecommendedCommandName)),
 		debug.NewCmdDebug(debug.RecommendedCommandName, util.GetFullName(fullName, debug.RecommendedCommandName)),
+
 		pipelines.NewCmdComponent(pipelines.RecommendedCommandName, util.GetFullName(fullName, pipelines.RecommendedCommandName)),
 		pipelines.NewCmdInit(pipelines.InitRecommendedCommandName, util.GetFullName(fullName, pipelines.InitRecommendedCommandName)),
+		environment.NewCmdAddEnv(environment.AddEnvRecommendedCommandName, util.GetFullName(fullName, environment.AddEnvRecommendedCommandName)),
```

Comment:
We can more NewCmdAddEnv() to environment.NewCmdEnv().   See https://github.com/rhd-gitops-example/odo/blob/3022d3edaa42b9173f09c99d69130a0b7048c5f0/pkg/odo/cli/preference/preference.go#L23 as an example.

Reply:
moved NewCmdAddEnv() to environment.NewCmdEnv().